recipe/meta.yaml

```diff
@@ -24,13 +24,14 @@ requirements:
     - wheel
     - setuptools
   run:
-    - python
+    - python >=3.7
```

Comment:
This is not a noarch package. The run constraint should not express a python version constraint.

Reply:
If we'll rebuild it as **noarch** we can keep it as it is

Reply:
Currently the package is a noarch on conda-forge, do we want to have the package reflect the upstream? or do we want to keep it as it was before?
https://github.com/conda-forge/dropbox-feedstock/blob/master/recipe/meta.yaml#L15

Reply:
The `python` constraint should match the versions supported upstream, not the latest version in the Anaconda repositories.

Reply:
This is a `noarch: python` package. There's no reason to constrain its runtime Python requirements to the versions actively supported by Anaconda.